package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "defender-remix-plugin-sveltekit",
-	"version": "0.0.1",
+	"version": "0.0.2",
```

I don't think should bump this version since we don't publish this package to NPM.  Rather, I think we should mark it as `private` so it cannot get published.  cc @MCarlomagno 
```suggestion
	"private": true,
```